### ISSUE TITLE
dashboard_info.js Error 400

### ISSUE THREAD

**Issue by kjellgren** [netdata/netdata#9787]
##### Bug report summary
Dashboard display :
Cannot load required JS library: http://myserver:19999/dashboard_info_custom.js

F12/Network in in the browser show that :
"Requested URL: http://myserver:19999/http://myserver:19999/dashboard_info_custom.js"
(base URL twice)

 http://myserver:19999/dashboard_info_custom.js load fine in the browser

##### OS / Environment
RHEL (RedHat) 7.6 x86_64

##### Netdata version
netdata v1.24.0

##### Component Name
web

##### Steps To Reproduce
In /etc/netdata.conf
```
[web]
   custom dashboard_info.js = dashboard_info_custom.js
```

##### Browser
Firefox Quantum 68.11.0esr (64-bit) on CentOS

**Comment by jacekkolasa** [comment]
Hi @kjellgren, can you write what you have in the response from `http://myserver:19999/api/v1/charts`, in `"custom_info"` field?

**Comment by kjellgren** [comment]
Hi @jacekkolasa 

> what you have in the response from http://myserver:19999/api/v1/charts, in "custom_info" field?

custom_info: "dashboard_info_custom.js"

**Comment by kjellgren** [comment]
Hi @jacekkolasa 

> what you have in the response from http://myserver:19999/api/v1/charts, in "custom_info" field?

custom_info: "dashboard_info_custom.js"

**Comment by kjellgren** [comment]
Hi @jacekkolasa 

> what you have in the response from http://myserver:19999/api/v1/charts, in "custom_info" field?

custom_info: "dashboard_info_custom.js"

**Comment by kjellgren** [comment]
Hi @jacekkolasa 

> what you have in the response from http://myserver:19999/api/v1/charts, in "custom_info" field?

custom_info: "dashboard_info_custom.js"

**Comment by kjellgren** [comment]
Hi @jacekkolasa 

> what you have in the response from http://myserver:19999/api/v1/charts, in "custom_info" field?

custom_info: "dashboard_info_custom.js"

**Comment by kjellgren** [comment]
Hi @jacekkolasa 

> what you have in the response from http://myserver:19999/api/v1/charts, in "custom_info" field?

custom_info: "dashboard_info_custom.js"

**Comment by jacekkolasa** [comment]
it's hard for me to reproduce it. Can you confirm that you see the version with new nav bar/sidebar, like here: https://frankfurt.my-netdata.io/ ?

**Comment by kjellgren** [comment]
Sorry forgotten a important pice of information. I am talking about the old dashboard  http://myserver:19999/old
And I have installed Netdata using RPMs
```
# rpm -qa | fgrep netdata
netdata-1.24.0-1.el7.x86_64
netdata-plugin-freeipmi1.24.0-1.el7.x86_64
#
```

**Comment by jacekkolasa** [comment]
Indeed, the problem exists in old dashboard. I've pushed a fix. Thanks!
